src/components/Game.tsx

```diff
@@ -1,44 +1,26 @@
-import * as React from "react";
-import { play, GESTURES, Gesture, Result } from "../game/game";
+import React, { useEffect, useState } from "react";
```

Comment:
can be: 
`import React, { FunctionComponent, useEffect, useState } from "react";`